bindings/cpp/logger.cpp

```diff
@@ -137,13 +136,12 @@ void dnet_node_set_trace_id(dnet_logger *logger, uint64_t trace_id, int tracebit
 	if (scoped_count >= DNET_SCOPED_LIMIT) {
 		dnet_log_only_log(logger, DNET_LOG_ERROR,
 			"logic error: you must not call dnet_node_set_trace_id twice, dnet_node_unset_trace_id call missed");
-		++scoped_count;
+		// crash
+		char *ptr = NULL;
+		*ptr = 1;
```

Comment:
Increasing scope here is a bug actually, although it never reached that level in our code.
Which I found by changing `DNET_SCOPED_LIMIT` and looking for crash in tests.
I'm against asserts (this is clearly a simplified version of assert) and this code can easily continue working without assertion, so I'd better revert this commit